sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/README.md

```diff
@@ -0,0 +1,47 @@
+# Azure ProgrammableConnectivity client library for .NET
+
+This SDK allows .NET applications to interact with Azure Programmable Connectivity Gateway, instead of using `curl` requests. For information on APC, see [docs](https://learn.microsoft.com/en-us/azure/programmable-connectivity/).
```

Comment:
The "Azure SDK" is the all-up branding for the collection of client and management libraries.   Please use "client library" to refer to this specific library.

Reply:
Have changed to `This client library allows .NET`...